### PR TITLE
Expose more parameters in the class interface

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

File: include/itkANTSRegistration.h [hunk 1]
```diff
@@ -121,6 +127,14 @@ class ANTSRegistration : public ProcessObject
   itkSetStringMacro(TypeOfTransform);
   itkGetStringMacro(TypeOfTransform);
 
+  /** The metric for the affine part (GC, mattes, meansquares). */
```
Comment (thewtex) [inline include/itkANTSRegistration.h:130]: Please enumerate the possible values in the docstring

File: include/itkANTSRegistration.h [hunk 2]
```diff
@@ -121,6 +127,14 @@ class ANTSRegistration : public ProcessObject
...
+  itkSetStringMacro(AffineMetric);
+  itkGetStringMacro(AffineMetric);
+
+  /** The metric for the SyN part (CC, mattes, meansquares, demons). */
```
Comment (thewtex) [inline include/itkANTSRegistration.h:134]: Please enumerate the possible values in the docstring

**Review by thewtex — CHANGES_REQUESTED** [review]
Looking good.

Documentation request inline.

Needs more tests.

**Comment by dzenanz** [comment]
> Needs more tests.

Passing some of the values in tests breaks some of the tests. So I was putting off test expansion until I most of the parameters exposed. That is now the next step.

I will enumerate possible values for the metric.

File: include/itkANTSRegistration.h (hunk 1 above)
Comment (dzenanz) [reply include/itkANTSRegistration.h:130]: Waiting on resolution of this: https://github.com/ANTsX/ANTs/issues/1668

**Review by thewtex — APPROVED** [review]

**Pull request merged**